spec/workers/receive_salmon_spec.rb

```diff
@@ -12,7 +12,7 @@
       end
     }
   end
-  it 'calls receive_salmon' do
+  it 'calls receive' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.